docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/configuring-product-visibility-by-site-using-channels.md

```diff
@@ -1,6 +1,6 @@
-# Configuring Product Visibility by Site Using Channels
+# Configuring Product Visibility by Channel
 
-By default in Liferay Commerce Products added to any catalog are visible globally - products are visible across all sites added to a Liferay Commerce installation. Store administrators may want to limit product visibility to particular channels based on their criteria.
```

Comment:
"Based on their criteria" is not a particularly helpful piece of information and therefore suggested to be removed.